sections/main-password-header.liquid

```diff
@@ -1,15 +1,21 @@
+<style type="text/css">
+  .password-logo {
+    max-width: {{ section.settings.logo_max_width }}px
+  }
+</style>
+
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="password-header">
     {%- if section.settings.logo -%}
-      <img
-        src="{{ section.settings.logo | image_url: width: 500, height: 500 }}"
-        class="password-logo"
-        alt="{{ section.settings.logo.alt | default: shop.name | escape }}"
-        style="max-width: {{ section.settings.logo_max_width }}px"
-        width="{{ section.settings.logo_max_width }}"
-        height="{{ section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio }}"
-        loading="lazy"
-      >
+      {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
+      {% assign logo_height =  section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio %}
+      {{ section.settings.logo | image_url: width: section.settings.logo.width | image_tag: 
+        loading: 'lazy', 
+        class: 'password-logo', 
+        width: section.settings.logo_max_width,
+        height: logo_height,
+        alt: logo_alt
+      }}
```

Comment:
```suggestion
      {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
      {%- assign logo_height = section.settings.logo_max_width | divided_by: section.settings.logo.aspect_ratio -%}
      {{ section.settings.logo | image_url: width: section.settings.logo.width | image_tag:
        loading: 'lazy',
        class: 'password-logo',
        widths: '50, 100, 150, 200, 250, 300, 400, 500',
        width: section.settings.logo_max_width,
        height: logo_height,
        alt: logo_alt
      }}
```

One thing I noticed was that for the Password logo, without `widths: '50, 100, 150, 200, 250, 300, 400, 500',`, the logo appeared slightly blurred. For consistency, I think it makes sense to include this since the header has it too 🤔

Reply:
Good catch Andrew 👍 Missed this one